cmd/client_changefeed.go

```diff
@@ -359,10 +355,12 @@ func verifyChangefeedParamers(ctx context.Context, cmd *cobra.Command, isCreate
 		CreatorVersion:    version.ReleaseVersion,
 	}
 
+	// user is not allowed to set sort-dir at changefeed level
 	if sortDir != "" {
-		cmd.Printf("[WARN] --sort-dir is deprecated in changefeed settings. " +
-			"Please use `cdc server --sort-dir` if possible. " +
-			"The sort-dir here will be no-op\n")
+		cmd.Printf(color.HiYellowString("[WARN] --sort-dir is deprecated in changefeed settings. " +
```

Comment:
Can you test whether `color.HiYellowString` will cause any unexpected output when stdout is something OTHER THAN an ANSI terminal?

Reply:
it's documented